prodtests/full-system-test/run-workflow-on-inputlist.sh

```diff
@@ -64,11 +69,19 @@ else
   echo "ERROR: Unsupported mode $1 requested"
   exit 1
 fi
-start_process $MYDIR/dpl-workflow.sh
+
+if [ -f dpl-workflow.sh ]; then
+    echo "Use dpl-workflow.sh macro passed as input"
+else
+    echo "Use dpl-workflow.sh macro from ${MYDIR}"
+    cp $O2_ROOT/prodtests/full-system-test/dpl-workflow.sh .
+fi
+start_process ./dpl-workflow.sh
```

Comment:
As discussed privately, I would prefer to have an explicit setting via an env variable to set a directory for the dpl-workflow.sh script, instead of taking one from the local directory.

Reply:
OK, after thinking about this again, this is actually dangerous. You are creating a local copy of the script, and afterwards you will be using the local copy if present. I.e., if you run once, then you update O2, and then you run again in the same folder, it will use the old version of dpl-workflow.sh. I think we should not do this. This will lead to confusion.

#7606 is a proposal how we could add this feature, does this work for you? Btw, `start_process` will anyway print `Started $@ with PID $PID` with the full path, so I didn't add an additional echo.

Reply:
Btw, concerning multiple running in the same folder: the AOD production is requested in update mode and fails if AO2D.root is found in the folder. Either update should be suppressed or the file should removed before starting the job.